functions/ocr/app/ocr_test.go

```diff
@@ -38,28 +38,21 @@ const (
 )
 
 var (
-	projectID       string
 	bucketName      string
 	imageBucketName string
 )
 
-// TestMain sets up the config rather than using the config file
-// which contains placeholder values.
 func setupTests(t *testing.T) {
 	ctx := context.Background()
 	projectID = os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
 	if projectID == "" {
 		t.Skip("GOLANG_SAMPLES_PROJECT_ID is unset")
 	}
-	bucketName = fmt.Sprintf("%s-result", projectID)
+	resultBucket = fmt.Sprintf("%s-result", projectID)
```

Comment:
Confirming you tried deploying and running this?

Reply:
I had, but did not realized it skiped before locally. Tested, working now.